web/logic.py

```diff
@@ -1,8 +1,10 @@
-from typing import List
+import logging
+from typing import List, Tuple, Optional
 
 import pandas as pd
 from naïve_bayes.sentiment_analysis.model import SentimentAnalysisModel
 from naïve_bayes.tweet_disaster_classification.model import NaiveBayesModel
+from models import db, Project, Category
 
 
 def make_sentiment_prediction(text) -> pd.Series:
```

Comment:
Add dependency injection so that the method takes in a model as a parameter instead of initializing it and importing weights every time the method is called